src/main/java/io/strimzi/kafka/quotas/StaticQuotaConfig.java

```diff
@@ -35,6 +36,7 @@ public StaticQuotaConfig(Map<String, ?> props, boolean doLog) {
                         .define(REQUEST_QUOTA_PROP, DOUBLE, Double.MAX_VALUE, HIGH, "Request processing time quota (in seconds)")
                         .define(STORAGE_QUOTA_SOFT_PROP, LONG, Long.MAX_VALUE, HIGH, "Hard limit for amount of storage allowed (in bytes)")
                         .define(STORAGE_QUOTA_HARD_PROP, LONG, Long.MAX_VALUE, HIGH, "Soft limit for amount of storage allowed (in bytes)")
+                        .define(STORAGE_QUOTA_PERCENTS_PROP, STRING, "/tmp/kafka-logs:80:100", HIGH, "Percents, soft and hard, allowed for volumes (FileStore) - in format path1:soft1:hard1|path2:soft2:hard2|...")
```

Comment:
```suggestion
                        .define(STORAGE_QUOTA_PERCENTS_PROP, STRING, "/tmp/kafka-logs:80:100", HIGH, "Per-log dir percentage soft and hard limits allowed for volumes (FileStore) - in the format 'logdir1:soft1:hard1|logdir2:soft2:hard2|...' with sort and hard limits in the range [1, 100].")
```

Also: there's a version of `define` that takes a `Validator`, perhaps we should use that here with a regex-based validator.